src/features/events/rpc/getAllEvents.ts

```diff
@@ -20,22 +20,25 @@ export const getAllEventsDef = {
 export default makeRPCDef<Params, Result>(getAllEventsDef.name);
 
 async function handle(params: Params, apiClient: IApiClient): Promise<Result> {
-  const memberships = await apiClient.get<ZetkinMembership[]>(
-    '/api/users/me/memberships'
-  );
+  const memberships = await apiClient
+    .get<ZetkinMembership[]>('/api/users/me/memberships')
+    .then((memberships) =>
+      memberships.filter((membership) => membership.follow)
+    );
```

Comment:
Nice solve of the bug! 🐛 

In the Zetkin code we prefer to no use `.then(...)` but instead `await` something, and then use that thing on the next line. We find it more easy to read and maintain when the action of fetching something is separated from the action of handling that data. 

So, I wonder if you would want to change this to make it in this style instead: 
```
const allMemberships = await apiClient.get...() ;
const filteredMemberships = allMemberships.filter(...);
```